src/MCPClient/install/README.md

```diff
@@ -1,13 +1,190 @@
+    - **Type:** `string`
+    - :red_circle: **Mandatory!**
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_SHAREDDIRECTORYMOUNTED`**:
+    - **Description:** location of the Archivematica Shared Directory.
+    - **Type:** `string`
+    - **Default:** `/var/archivematica/sharedDirectory/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_PROCESSINGDIRECTORY`**:
+    - **Description:** location of the Archivematica Currently Procesing Directory.
+    - **Type:** `string`
+    - **Default:** `/var/archivematica/sharedDirectory/currentlyProcessing/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_REJECTEDDIRECTORY`**:
+    - **Description:** location of the Archivematica Rejected Directory.
+    - **Type:** `string`
+    - **Default:** `%%sharedPath%%rejected/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_WATCHDIRECTORYPATH`**:
+    - **Description:** location of the Archivematica Watched Directory.
+    - **Type:** `string`
+    - **Default:** `/var/archivematica/sharedDirectory/watchedDirectories/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_CLIENTSCRIPTSDIRECTORY`**:
+    - **Description:** location of the client scripts directory.
+    - **Type:** `string`
+    - **Default:** `/usr/lib/archivematica/MCPClient/clientScripts/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_CLIENTASSETSDIRECTORY`**:
+    - **Description:** location of the client assets directory.
+    - **Type:** `string`
+    - **Default:** `/usr/lib/archivematica/MCPClient/assets/`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_LOADSUPPORTEDCOMMANDSSPECIAL`**:
+    - **Description:** enables loading special modules.
+    - **Type:** `boolean`
+    - **Default:** `true`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_MCPARCHIVEMATICASERVER`**:
+    - **Description:** address of the Gearman server.
+    - **Type:** `string`
+    - **Default:** `localhost:4730`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_NUMBEROFTASKS`**:
+    - **Description:** number of Gearman workers that the process will run in parallel. When `0`, MCPClient will start as many workers as CPU cores are found in the system.
+    - **Type:** `int`
+    - **Default:** `0`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_ARCHIVEMATICACLIENTMODULES`**:
+    - **Description:** location of the client modules configuratino file. This can be useful when the user wants to set up workers that can only work in a limited number of tasks, e.g. a worker exclusively dedicated to antivirus scanning or file identification.
+    - **Type:** `string`
+    - **Default:** `/usr/lib/archivematica/MCPClient/archivematicaClientModules`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_ELASTICSEARCHSERVER`**:
+    - **Description:** address of the Elasticsearch server.
+    - **Type:** `string`
+    - **Default:** `localhost:9200`
+
+- **`ARCHIVEMATICA_MCPCLIENT_MCPCLIENT_ELASTICSEARCHTIMEOUT`**:
+    - **Description:** configures the Elasticsearch client to stop waiting for a response after a given number of seconds.
+    - **Type:** `float`
+    - **Default:** `localhost:9200`
```

Comment:
This should be a value in seconds, not a host and port (value for ELASTICSEARCHTIMEOUT)

Reply:
Good catch, thanks, fixed!